src/Classes/ServiceAPI/MyRadio_Podcast.php

```diff
@@ -403,6 +403,12 @@ public static function create(
         MyRadio_Show $show = null,
         $credits = null
     ) {
+        //Explode the tags
+        $tags = explode(' ', $tags);
```

Comment:
Shouldn't this use CoreUtils::explodeTags?

Reply:
I think coreUtils::explodeTags was created after this, and supersedes this PR, closing 👍